kod/object.kod

```diff
@@ -685,6 +685,11 @@ messages:
    {
       return;
    }
+   
+   UserEntered(who = $)
```

Comment:
Since this is the highest place in the hierarchy this appears, there should be documentation here.

Also, I guess this could be on ActiveObject rather than Object, since Room iterates over its active objects list to make the call.  I'm not sure why all of these similar functions wound up on Object instead.